ush/global_cycle.sh

```diff
@@ -263,7 +264,9 @@ use_ufo=${use_ufo:-.true.}
 DONST=${DONST:-"NO"}
 DO_SFCCYCLE=${DO_SFCCYCLE:-.true.}
 DO_LNDINC=${DO_LNDINC:-.false.}
```

Comment:
DO_SOI_INC_GSI is not defined in the prolog.

Reply:
Hi George, thanks for the question. What do you mean by "prolog"? DO_SOI_INC_GSI along with other two options were defined in the ush/global_cycle.sh from Lines 267-269:

267 DO_SOI_INC_GSI=${DO_SOI_INC_GSI:-.false.}
268 DO_SNO_INC_JEDI=${DO_SNO_INC_JEDI:-.false.}
269 DO_SOI_INC_JEDI=${DO_SOI_INC_JEDI:-.false.}

Reply:
Define it after line 144.

Reply:
Thanks! Done.